api/worker.js

```diff
@@ -1,7 +1,7 @@
 import console from './console.js'
 
 // eslint-disable-next-line no-new-func
-const GlobalWorker = new Function('return Worker')()
+const GlobalWorker = new Function('return this.Worker')()
```

Comment:
why not `globalThis`?

Reply:
Because typescript 😑😅

Reply:
no :( we can't use the webworker lib without conflicts